examples/additional-overlays/choropleth-overlay.js

```diff
@@ -114,26 +113,38 @@ export default class ChoroplethOverlay extends Component {
   }
 
   render() {
-    const {viewport: {width, height}} = this.context;
-    const pixelRatio = window.devicePixelRatio || 1;
     return (
-      <canvas
-        ref="overlay"
-        width={ width * pixelRatio }
-        height={ height * pixelRatio }
-        style={ {
-          width: `${width}px`,
-          height: `${height}px`,
-          position: 'absolute',
-          pointerEvents: 'none',
-          opacity: this.props.globalOpacity,
-          left: 0,
-          top: 0
-        } }/>
+      createElement(StaticContext.Consumer,
+        null, sContext => {
```

Comment:
Why not jsx here? Looks inconsistent.

Reply:
Legacy build system. I will upgrade it in another PR.

Reply:
Hmm didn't see that it was an example. Examples already can transpile JSX. There should not be any usage of context components on the application level.